### ISSUE TITLE
`CreatureBehavior#panic()` doesn't work on 1.16

### ISSUE THREAD

**Issue by datatags** [GamerCoder215/MobChip#34]
### Bug Type

Compatibility Issues, Console Errors

### Description

`CreatureBehavior#panic()` doesn't work when using MobChip on v1_16_R2 and v1_16_R3, giving a console error. (Lower versions are not affected because behaviors aren't supported on those versions anyway.)

### Expected Behavior

The creature to run around randomly as if it had been punched.

### Actual Behavior

The animal doesn't do anything, and this message is printed to console:
```
[20:59:10] [Server thread/ERROR]: java.lang.ClassNotFoundException: net.minecraft.server.v1_16_R3.AnimalPanic
[20:59:10] [Server thread/ERROR]:     org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:138)
[20:59:10] [Server thread/ERROR]:     org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99)
[20:59:10] [Server thread/ERROR]:     java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
[20:59:10] [Server thread/ERROR]:     java.base/java.lang.Class.forName0(Native Method)
[20:59:10] [Server thread/ERROR]:     java.base/java.lang.Class.forName(Class.java:375)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.shaded.mobchip.abstraction.ChipUtil1_16_R3.runBehavior(ChipUtil1_16_R3.java:423)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.shaded.mobchip.abstraction.ChipUtil1_16_R3.runBehavior(ChipUtil1_16_R3.java:396)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.shaded.mobchip.bukkit.BukkitEntityBehavior.run(BukkitEntityBehavior.java:29)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.shaded.mobchip.bukkit.BukkitCreatureBehavior.panic(BukkitCreatureBehavior.java:24)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.shaded.mobchip.ai.behavior.CreatureBehavior.panic(CreatureBehavior.java:28)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.cosmetics.gadgets.GadgetExplosiveSheep$SheepColorRunnable.lambda$run$0(GadgetExplosiveSheep.java:128)
[20:59:10] [Server thread/ERROR]:     be.isach.ultracosmetics.util.EntitySpawner.run(EntitySpawner.java:65)
[20:59:10] [Server thread/ERROR]:     org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:81)
[20:59:10] [Server thread/ERROR]:     org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:400)
[20:59:10] [Server thread/ERROR]:     net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1179)
[20:59:10] [Server thread/ERROR]:     net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:394)
[20:59:10] [Server thread/ERROR]:     net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1127)
[20:59:10] [Server thread/ERROR]:     net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:966)
[20:59:10] [Server thread/ERROR]:     net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:273)
[20:59:10] [Server thread/ERROR]:     java.base/java.lang.Thread.run(Thread.java:831)
```

**Comment by GamerCoder215** [comment]
Can confirm happens on 1.16, and on future versions as well.

The internal methods use reflection to look for the Behavior constructor, instead of constructing it manually on individual versions.
The Reflection was looking for a mojang-mapped name instead of a spigot-mapped name that is present on the server.

**Comment by GamerCoder215** [comment]
Update:
As it turns out, the Behavior doesn't actually exist until 1.17; I didn't notice that previously until I inspected the 1.16 `BehaviorPanic` behavior.

I'll probably recreate it for 1.16.5 and below.

**Comment by GamerCoder215** [comment]
A Draft fix is available on [`d3a8322`](https://github.com/GamerCoder215/MobChip/commit/d3a83224d4ac1b3886a1062191ba1dd69035e439), but this requires additional testing.

I'm unsure if the Memory that determines whether a Mob is currently panicking has any visual effects, but the recreated behavior should have the same functionality as the vanilla one on 1.17+.

I'll leave this issue open until testing is done and has positive results.

**Comment by GamerCoder215** [comment]
Behavior works ok, however:

Behaviors will only properly run if they have certain memories present and/or not present. I'll work on documentation on what behavior memories are required in order to properly work.

Closing as completed.
